content/src/main/content/jcr_root/apps/acs-commons/authoring/dam-copy-publishurl/.content.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="dam.copy.publishurl"/>
```

Comment:
we typically prefix our clientlibs with "acs-aem-commons" to namespace them; its unlikely someone else would use "dam.copy.publishurl" but not entirely unreasonable. Check out some of the similar clientlibs for the CL naming conventions.

Reply:
I think this should be added to one of the existing clientlibs used on this page already. That way, the `head` node doesn't need to be overlayed.

Reply:
`dam.gui.coral.metadataeditor` perhaps